lib/ex_docker_build/api/docker_remote_api.ex

```diff
@@ -179,4 +179,25 @@ defmodule ExDockerBuild.API.DockerRemoteAPI do
     |> URI.to_string()
     |> HTTPoison.get()
   end
+
+  @impl Docker
+  def get_archive(container_id, path) do
```

Comment:
`get_archive` and `extract_archive` are the same function

Reply:
they are similar, but one makes a get request and the other one a post